package.json

```diff
@@ -13,7 +13,8 @@
     "lint": "ng lint",
     "lint:styles": "stylelint \"src/**/*.scss\"",
     "cypress": "cypress run --browser chrome",
-    "cypress-open": "cypress open",
+    "cypress:local": "cypress run --browser chrome --config baseUrl=http://localhost:4200",
+    "cypress:open": "cypress open --config baseUrl=http://localhost:4200",
```

Comment:
I couldn't see from a quick DDG but am used to tools where the command line takes precedence over ENV_VARs & config files.  Please let me know / update as needed